### PR TITLE
Update Azure Policy Add-on images

### PR DESCRIPTION
**Pull request by RamyasreeChakka**
Removing the old images and adding new images for Azure Policy Add-on.

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -530,13 +530,13 @@ done
 
 ADDON_IMAGES="
 mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v2.0.1
-mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0-beta.8
+mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0
```
Comment (robbiezhang) [inline vhdbuilder/packer/install-dependencies.sh:533]: can you move it down below the beta.12? we assume the lower ones are newer

File: vhdbuilder/packer/install-dependencies.sh [hunk 2]
```diff
@@ -530,13 +530,13 @@ done
...
 mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0-beta.12
 mcr.microsoft.com/oss/kubernetes/external-dns:v0.6.0-hotfix-20200228
 mcr.microsoft.com/oss/kubernetes/defaultbackend:1.4
 mcr.microsoft.com/oss/kubernetes/ingress/nginx-ingress-controller:0.19.0
 mcr.microsoft.com/oss/virtual-kubelet/virtual-kubelet:1.2.1.1
-mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod:prod_20200519.1
+mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod:prod_20200901.1
```
Comment (robbiezhang) [inline vhdbuilder/packer/install-dependencies.sh:539]: same ask here.

File: vhdbuilder/packer/install-dependencies.sh (hunk 1 above)
Comment (RamyasreeChakka) [reply vhdbuilder/packer/install-dependencies.sh]: @robbiezhang Thanks, updated the order.

File: vhdbuilder/packer/install-dependencies.sh (hunk 2 above)
Comment (RamyasreeChakka) [reply vhdbuilder/packer/install-dependencies.sh]: done, thanks

**Pull request merged**